packages/firestore/src/api/timestamp.ts

```diff
@@ -92,4 +94,21 @@ export class Timestamp {
       ')'
     );
   }
+
+  // Overriding valueOf() allows Timestamp objects to be compared in JavaScript using the
```

Comment:
Nit: The top part should be a JSDoc-style method comment.

Reply:
Done